### ISSUE TITLE
Add prometheus support 

### ISSUE THREAD

**Issue by imraan-go** [uptrace/bun#211]
Looking at the source code of bunotel, looks like it only support traces. Do you have any plan to support metrics exporting perhaps via opentelemetry? I want to export http metrics along with bun metrics to prometheus.

**Comment by vmihailenco** [comment]
Yes, bunotel will support metrics. The rough estimation is next week.

**Comment by imraan-go** [comment]
Any example usage with Treemux router is appreciated. Since metrics collection documentation is missing in the official bun website. @vmihailenco

**Comment by vmihailenco** [comment]
Well, all you should do is to configure OpenTelemetry to export metrics to Prometheus. I will write an article at blog.uptrace.dev for SEO purposes :), but for now you can try to figure that for yourself - https://github.com/open-telemetry/opentelemetry-go/blob/main/example/prometheus/main.go . The only treemux specific line is `http.HandleFunc("/", exporter.ServeHTTP)` which can be written as:

````go
-http.HandleFunc("/", exporter.ServeHTTP)
+router.Compat().GET("/", exporter.ServeHTTP)
````

After that you need to visualize the data somehow. With Uptrace it looks something like this

![bun-metrics](https://user-images.githubusercontent.com/290976/135253876-7aa80f3c-380b-4c22-b8ae-c242a38683e8.png)

The histogram accuracy (p50) is really bad, but it will be improved with exponential histograms - github.com/open-telemetry/opentelemetry-go/pull/2261.
